avaframe/in3Utils/geoTrans.py

```diff
@@ -30,6 +31,10 @@ def projectOnRaster(dem, Points, interp='bilinear'):
         Points dictionary (x,y)
     interp: str
         interpolation option, between nearest or bilinear
+    what: str
+        which key in the dem dict should be used for the interpolation?
+    where: str
+        which key in the Points dict should be updated with the interpolated data?
```

Comment:
mh, not sure whether what/where are the best names. from/to ? source/target? sourceKey/targetKey?